```
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02832/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-04-08-2300/bin/el8_amd64_gcc12/cmsDriver.py", line 40, in <module>
    run()
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02832/el8_amd64_gcc12/cms/cmssw/CMSSW_14_1_X_2024-04-08-2300/bin/el8_amd64_gcc12/cmsDriver.py", line 16, in run
    configBuilder.prepare()
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02832/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_X_2024-04-09-1100/src/Configuration/Applications/python/ConfigBuilder.py", line 2298, in prepare
    self.addStandardSequences()
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02832/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_X_2024-04-09-1100/src/Configuration/Applications/python/ConfigBuilder.py", line 826, in addStandardSequences
    getattr(self,"prepare_"+stepName)(stepSpec = getattr(self,stepName+"DefaultSeq"))
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02832/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_X_2024-04-09-1100/src/Configuration/Applications/python/ConfigBuilder.py", line 1598, in prepare_L1
    self.loadAndRemember(self.L1EMDefaultCFF)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02832/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_X_2024-04-09-1100/src/Configuration/Applications/python/ConfigBuilder.py", line 354, in loadAndRemember
    self.process.load(includeFile)
  File "/cvmfs/cms-ib.cern.ch/sw/x86_64/nweek-02832/el8_amd64_gcc12/cms/cmssw-patch/CMSSW_14_1_X_2024-04-09-1100/src/FWCore/ParameterSet/python/Config.py", line 762, in load
    module = __import__(moduleName)
  File "/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/44677/38719/CMSSW_14_1_X_2024-04-09-1100/src/Configuration/StandardSequences/python/SimL1Emulator_cff.py", line 3, in <module>
    from L1Trigger.Configuration.SimL1Emulator_cff import *
  File "/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/44677/38719/CMSSW_14_1_X_2024-04-09-1100/src/L1Trigger/Configuration/python/SimL1Emulator_cff.py", line 124, in <module>
    from L1Trigger.L1CaloTrigger.l1tPhase2CaloJetEmulator_cff import *
ModuleNotFoundError: No module named 'L1Trigger.L1CaloTrigger.l1tPhase2CaloJetEmulator_cff'
```

One second, while I check this...

--- a/L1Trigger/Configuration/python/SimL1Emulator_cff.py
+++ b/L1Trigger/Configuration/python/SimL1Emulator_cff.py
@@ -2,7 +2,7 @@ import FWCore.ParameterSet.Config as cms
 
 # Defines the L1 Emulator sequence for simulation use-case subsystem emulators
 # run on the results of previous (in the hardware chain) subsystem emulator:
-#
+#  
 #     SimL1Emulator = cms.Sequence(...)
 #
 # properly configured for the current Era (e.g. Run1, 2015, or 2016).  Also
@@ -17,7 +17,7 @@ import FWCore.ParameterSet.Config as cms
 
 # Notes on Inputs:
 
-# ECAL TPG emulator and HCAL TPG run in the simulation sequence in order to be able
+# ECAL TPG emulator and HCAL TPG run in the simulation sequence in order to be able 
 # to use unsuppressed digis produced by ECAL and HCAL simulation, respectively
 # in Configuration/StandardSequences/python/Digi_cff.py
 # SimCalorimetry.Configuration.SimCalorimetry_cff
@@ -48,7 +48,7 @@ SimL1EmulatorCore = cms.Sequence(SimL1EmulatorCoreTask)
 SimL1EmulatorTask = cms.Task(SimL1EmulatorCoreTask)
 SimL1Emulator = cms.Sequence( SimL1EmulatorTask )
 
-#
+# 
 # Emulators are configured from DB (GlobalTags)
 #
 
@@ -63,7 +63,7 @@ _phase2_siml1emulator = SimL1EmulatorTask.copy()
 # ########################################################################
 # ########################################################################
 #
-# Phase-2
+# Phase-2 
 #
 # ########################################################################
 # ########################################################################
@@ -77,11 +77,11 @@ _phase2_siml1emulator.add(CalibratedDigis)
 from L1Trigger.DTTriggerPhase2.dtTriggerPhase2PrimitiveDigis_cfi import *
 _phase2_siml1emulator.add(dtTriggerPhase2PrimitiveDigis)
 
-# HGCAL TP
+# HGCAL TP 
 # ########################################################################
 from  L1Trigger.L1THGCal.hgcalTriggerPrimitives_cff import *
 _phase2_siml1emulator.add(L1THGCalTriggerPrimitivesTask)
-
+ 
 # ########################################################################
 # Phase 2 L1T
 # ########################################################################
@@ -111,7 +111,7 @@ l1tCaloJet = l1tCaloJetProducer.clone (
 # ----    Produce the simulated CaloJet HTT Sums
 from L1Trigger.L1CaloTrigger.l1tCaloJetHTTProducer_cfi import *
 l1tCaloJetHTT = l1tCaloJetHTTProducer.clone(
-    BXVCaloJetsInputTag = ("L1CaloJet", "CaloJets")
+    BXVCaloJetsInputTag = ("L1CaloJet", "CaloJets") 
 )
 # ----    Produce the NNCaloTau
 from L1Trigger.L1CaloTrigger.l1tNNCaloTauProducer_cfi import *
@@ -121,25 +121,13 @@ from L1Trigger.L1CaloTrigger.l1tNNCaloTauEmulator_cfi import *
 _phase2_siml1emulator.add(l1tNNCaloTauEmulator)
 
 # ---- Produce the emulated CaloJets and Taus
-from L1Trigger.L1CaloTrigger.l1tPhase2CaloJetEmulator_cff import *
+from L1Trigger.L1CaloTrigger.l1tPhase2CaloJetEmulator_cfi import *
 
 _phase2_siml1emulator.add(l1tTowerCalibration)
 _phase2_siml1emulator.add(l1tCaloJet)
 _phase2_siml1emulator.add(l1tCaloJetHTT)
-_phase2_siml1emulator.add(L1TCaloJetsTausTask)
+_phase2_siml1emulator.add(l1tPhase2CaloJetEmulator)
 
-# Overlap and EndCap Muon Track Finder
-# ########################################################################
-from L1Trigger.L1TMuonOverlapPhase2.simOmtfPhase2Digis_cfi import *
-_phase2_siml1emulator.add(simOmtfPhase2Digis)
-
-from L1Trigger.L1TMuonEndCapPhase2.simCscTriggerPrimitiveDigisForEMTF_cfi import *
-from L1Trigger.L1TMuonEndCapPhase2.rpcRecHitsForEMTF_cfi import *
-from L1Trigger.L1TMuonEndCapPhase2.simEmtfDigisPhase2_cfi import *
-
-_phase2_siml1emulator.add(simCscTriggerPrimitiveDigisForEMTF)
-_phase2_siml1emulator.add(rpcRecHitsForEMTF)
-_phase2_siml1emulator.add(simEmtfDigisPhase2)
 
 # ########################################################################
 # Phase-2 L1T - TrackTrigger dependent modules
@@ -162,27 +150,19 @@ _phase2_siml1emulator.add(l1tVertexFinderEmulator)
 # Emulated GMT Muons (Tk + Stub, Tk + MuonTFT, StandaloneMuon)
 # ########################################################################
 from L1Trigger.Phase2L1GMT.gmt_cfi  import *
-l1tStubsGmt = gmtStubs.clone()
-l1tKMTFMuonsGmt = gmtKMTFMuons.clone(
-  stubs      = cms.InputTag('l1tStubsGmt','kmtf'),
+l1tTkStubsGmt = l1tGMTStubs.clone()
+l1tTkMuonsGmt = l1tGMTMuons.clone(
+    srcStubs  = 'l1tTkStubsGmt'
 )
-l1tFwdMuonsGmt = gmtFwdMuons.clone(
-    stubs  = 'l1tStubsGmt:tps'
-)
-l1tSAMuonsGmt = gmtSAMuons.clone(
-  barrelPrompt      = cms.InputTag('l1tKMTFMuonsGmt:prompt'),
-  barrelDisp        = cms.InputTag('l1tKMTFMuonsGmt:displaced'),
-  forwardPrompt     = cms.InputTag('l1tFwdMuonsGmt:prompt'),
-  forwardDisp     = cms.InputTag('l1tFwdMuonsGmt:displaced')
-)
-l1tTkMuonsGmt = gmtTkMuons.clone(
-    srcStubs  = 'l1tStubsGmt:tps'
-)
-_phase2_siml1emulator.add( l1tStubsGmt )
-_phase2_siml1emulator.add( l1tKMTFMuonsGmt )
-_phase2_siml1emulator.add( l1tFwdMuonsGmt )
-_phase2_siml1emulator.add( l1tSAMuonsGmt )
+l1tSAMuonsGmt = l1tStandaloneMuons.clone()
+_phase2_siml1emulator.add( l1tTkStubsGmt )
 _phase2_siml1emulator.add( l1tTkMuonsGmt )
+_phase2_siml1emulator.add( l1tSAMuonsGmt )
+
+## fix for low-pt muons, this collection is a copy of the l1tTkMuonsGmt collection 
+## in which we only keep those low pt muons with an SA muon associated to it. 
+l1tTkMuonsGmtLowPtFix = l1tGMTFilteredMuons.clone()
+_phase2_siml1emulator.add( l1tTkMuonsGmtLowPtFix )
 
 # Tracker Objects
 # ########################################################################
@@ -191,14 +171,11 @@ from L1Trigger.L1TTrackMatch.l1tTrackFastJets_cfi import *
 from L1Trigger.L1TTrackMatch.l1tTrackerEtMiss_cfi import *
 from L1Trigger.L1TTrackMatch.l1tTrackerHTMiss_cfi import *
 
-
-
 #Selected and Associated tracks for Jets and Emulated Jets
 _phase2_siml1emulator.add(l1tTrackSelectionProducerForJets)
 _phase2_siml1emulator.add(l1tTrackSelectionProducerExtendedForJets)
 _phase2_siml1emulator.add(l1tTrackVertexAssociationProducerForJets)
 _phase2_siml1emulator.add(l1tTrackVertexAssociationProducerExtendedForJets)
-
 
 #Selected and Associated tracks for EtMiss and Emulated EtMiss
 _phase2_siml1emulator.add(l1tTrackSelectionProducerForEtMiss)
@@ -206,14 +183,12 @@ _phase2_siml1emulator.add(l1tTrackSelectionProducerExtendedForEtMiss)
 _phase2_siml1emulator.add(l1tTrackVertexAssociationProducerForEtMiss)
 _phase2_siml1emulator.add(l1tTrackVertexAssociationProducerExtendedForEtMiss)
 
-
 #Track Jets, Track Only Et Miss, Track Only HT Miss
 _phase2_siml1emulator.add(l1tTrackJets)
 _phase2_siml1emulator.add(l1tTrackJetsExtended)
 _phase2_siml1emulator.add(l1tTrackFastJets)
 _phase2_siml1emulator.add(l1tTrackerEtMiss)
 _phase2_siml1emulator.add(l1tTrackerHTMiss)
-
 
 #Emulated Track Jets, Track Only Et Miss, Track Only HT Miss
 from L1Trigger.L1TTrackMatch.l1tTrackJetsEmulation_cfi import *
@@ -249,8 +224,8 @@ from L1Trigger.L1CaloTrigger.Phase1L1TJets_9x9trimmed_cff import *
 L1TPFJetsPhase1Task_9x9trimmed = cms.Task(  l1tPhase1JetProducer9x9trimmed, l1tPhase1JetCalibrator9x9trimmed, l1tPhase1JetSumsProducer9x9trimmed)
 _phase2_siml1emulator.add(L1TPFJetsPhase1Task_9x9trimmed)
 
-from L1Trigger.Phase2L1ParticleFlow.L1HPSPFTauProducer_cfi import *
-_phase2_siml1emulator.add(l1HPSPFTauEmuProducer)
+from L1Trigger.Phase2L1ParticleFlow.l1tHPSPFTauProducer_cfi import *
+_phase2_siml1emulator.add(l1tHPSPFTauProducer)
 
 # PF MET
 # ########################################################################

